### DISCUSSION TITLE
그림자 생성 도와주세요^^

### DISCUSSION THREAD

**Discussion by elimieum** [naverz/zepeto-studio-kor#222] [5. Q&A]
<font class="papago-parent"><font class="papago-source" style="display:none;">유니티 화면에서는 그림자가 생성이 되며 Game 창에서도 정상적으로 작동합니다.</font>유니티 화면에서는 그림자가 생성이 되며 Game 창에서도 정상적으로 작동합니다.</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>

</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>

</font><font class="papago-parent"><font class="papago-source" style="display:none;">다만, 큐알코드로 스캔해서 보면 아래의 이미지처럼 그림자 없이 나타납니다.</font>다만, 큐알코드로 스캔해서 보면 아래의 이미지처럼 그림자 없이 나타납니다.</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>

</font><font class="papago-parent"><font class="papago-source" style="display:none;">라이트 베이크도 해봤습니다.(Mixde mode)
</font>라이트 베이크도 해봤습니다.(Mixde mode)
</font><font class="papago-parent"><font class="papago-source" style="display:none;">유니티는 처음이라 어떻게 해야하는디 여기저기 정보검색을 해도 답이 없어
</font>유니티는 처음이라 어떻게 해야하는디 여기저기 정보검색을 해도 답이 없어
</font><font class="papago-parent"><font class="papago-source" style="display:none;">도움을 청해봅니다.</font>도움을 청해봅니다.</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>

</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>

</font>

**Comment by moondory77-naverz** [comment]
안녕하세요. 
올려주신 이미지가 첨부되지 않은 듯 합니다.
혹시 이미지를 다시한번 올려주실 수 있을까요?
감사합니다.

**Reply by elimieum** [reply]
몇일을 고민 했는데 
어찌어찌 하다보니 해결되었습니다.

질문에 답주시어 감사드립니다.
담에도 질문이 있으면 부탁드리겠습니다^~^

**Reply by vanni11** [reply]
동일한 증상으로 그림자가 표시 안되고있는데
혹시 해결 어떻게 하셨는지 알려주실 수 있을까요?

**Comment by hyeonjin-jo** [comment]
해결이 되신 것 같아서 완료 처리하겠습니다
